pvlib/bifacial/loss_models.py

```diff
@@ -0,0 +1,140 @@
+
+    Parameters
+    ----------
+    rmad : numeric
+        The Relative Mean Absolute Difference of the cell-by-cell total
+        irradiance. [Unitless]
+        Check out the *Notes* section for the equation to calculate it from the
+        bifaciality and the front and back irradiances.
+
+    model : float collection or numpy.polynomial.polynomial.Polynomial, default ``(0, 0.142, 0.032)``
+        The model coefficients to use.
+
+        If a :external:class:`numpy.polynomial.polynomial.Polynomial`,
+        it is evaluated as is. If not a ``Polynomial``, it must be the
+        coefficients of a polynomial in ``rmad``, where the first element is
+        the constant term and the last element is the highest order term. A
+        :external:class:`~numpy.polynomial.polynomial.Polynomial`
+        will be created internally.
+
+    fill_factor : float, optional
+        Fill factor at standard test condition (STC) of the module.
+        Accounts for different fill factors between the trained model and the
+        module under non-uniform irradiance.
+        If not provided, the model is used as is.
+
+    fill_factor_reference : float, default 0.79
+        Fill factor at STC of the module used to train the model.
+        If ``fill_factor`` is provided, this value is used to scale the output
+        of the model.
+
+    Returns
+    -------
+    loss : numeric
+        The fractional power loss. [Unitless]
+        Output will be a ``pandas.Series`` if ``rmad`` is a ``pandas.Series``.
+
+    Notes
+    -----
+    The default model implemented is equation (11) [1]_:
+
+    .. math::
+
+       M[\%] = 0.142 \Delta[\%] + 0.032 \Delta[\%]^2 \qquad & \text{(11)}
+
+    where :math:`\Delta[\%]` is the Relative Mean Absolute Difference of the
+    global irradiance, Eq. (4) of [1]_ and [2]_.
+
+    The losses definition is Eq. (1) of [1]_, and it's defined as a loss of the
+    output power:
+
+    .. math::
+
+       M[\%] = 1 - \frac{P_{Array}}{\sum P_{Cells}} \qquad \text{(1)}
+
+    To account for a module with a fill factor distinct from the one used to
+    train the model (``0.79`` by default), the output of the model can be
+    modified with Eq. (7):
+
+    .. math::
+
+       M[\%]_{FF_1} = M[\%]_{FF_0} \frac{FF_1}{FF_0} \qquad \text{(7)}
+
+    where parameter ``fill_factor`` is :math:`FF_1` and
+    ``fill_factor_reference`` is :math:`FF_0`.
+    In the section *See Also*, you will find two packages that can be used to
+    calculate the irradiance at different points of the module.
+
+    .. note::
+       The global irradiance RMAD is different from the backside irradiance
+       RMAD.
+
+    In case the RMAD of the backside irradiance is known, the global RMAD can
+    be calculated as follows, assuming the front irradiance RMAD is
+    negligible [2]_:
+
+    .. math::
+
+       RMAD(k \cdot X + c) = RMAD(X) \cdot k \frac{k \bar{X}}{k \bar{X} + c}
+       = RMAD(X) \cdot k \frac{1}{1 + \frac{c}{k \bar{X}}}
```

Comment:
I think it's a nice idea to guide users how to convert rearside RMAD into total RMAD, but it's not immediately obvious how to use this equation.  What do k, X, c represent?  Could we use symbols like `RMAD_{rear}`, `G_{front}` to make it clearer?